app/services/imports/lettings_logs_field_import_service.rb

```diff
@@ -453,5 +455,28 @@ def update_old_form_id(xml_doc)
         @logger.warn("lettings log with old id #{old_id} not found")
       end
     end
+
+    def update_postcode_full(xml_doc)
+      return if meta_field_value(xml_doc, "form-name").include?("Sales")
```

Comment:
Do we want an import task for sales logs too?

Reply:
same as above